src/auth/TokenService.ts

```diff
@@ -1,12 +1,26 @@
-import jwt from 'jsonwebtoken';
+// import jwt from 'jsonwebtoken';
 import User from '../user/User';
+import generator from '../shared/generator';
+import Token from './Token';
 
-const createToken = (user: User) => {
-  return jwt.sign({ id: user.id }, 'this-is-our-secret');
+const createToken = async (user: User) => {
+  // creating random string token and store in db under token field
+  const token = generator.randomString(32);
+
+  await Token.create({ token, userId: user.id as number });
+
+  return token;
+  // return jwt.sign({ id: user.id }, 'this-is-our-secret');
 };
 
-const verify = (token: string) => {
-  return jwt.verify(token, 'this-is-our-secret');
+const verify = async (token: string) => {
+  const tokenInDB = await Token.findOne({ where: { token: token } });
```

Comment:
Check if there's any logged in user with the same token in the database, if none then user is not logged in